internal/output/spdx_test.go

```diff
@@ -0,0 +1,72 @@
+package output_test
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/google/osv-scanner/v2/internal/cachedregexp"
+	"github.com/google/osv-scanner/v2/internal/output"
+	"github.com/google/osv-scanner/v2/internal/testutility"
+	"github.com/jedib0t/go-pretty/v6/text"
+)
+
+func normalizeSPDXOutput(t *testing.T, str string) string {
```

Comment:
I wonder if it's better to do the JSON matching, and then use the json normalization code.

I don't know if that's actually any better than this though, so happy to merge it as is.

Reply:
I don't think it makes much of a difference at this point, as I've already implemented it this way and it never felt like I was pushing things uphill - we can revisit if/when the output becomes more complicated